tests/unit/dataactcore/factories/job.py

```diff
@@ -119,3 +127,5 @@ class Meta:
     end_date = fuzzy.FuzzyDate(date(2010, 1, 1))
     block_certification = False
     message = fuzzy.FuzzyText()
+    application_type = fuzzy.FuzzyInteger(9999)
```

Comment:
When you add a SubFactory to a Factory, it's generally unnecessary to also add the ID you're using as the foreign key. Just pass it an object to use with the relationship and it'll figure out the rest